.github/workflows/e2e.yml

```diff
@@ -79,11 +79,15 @@ jobs:
             --image-replacements workspace/just.containerlookup \
             --namespace-file workspace/e2e.namespace \
             --platform ${{ inputs.platform }} \
-            --skip-undeploy="${{ inputs.skip-undeploy && 'true' || 'false' }}"
+            --skip-undeploy="true"
```

Comment:
Does this problem also affect https://github.com/edgelesssys/contrast/blob/f50e93b16f7cca03e0c8fd5aa70943567fe1d69c/.github/workflows/e2e_aks_runtime.yml#L90 ? 
If this is the case we need the same change over there. Also then the --skip-undeploy could be removed from the nix/Go side as it's always true (at least if no dev changes it locally in their justfile).

Reply:
Oh yeah, I forgot to remove it from the AKS runtime test, will do. I'll also remove skip-undeploy from the go side if you think that nobody needs that control locally